mesh_segmenter/package.xml

```diff
@@ -6,8 +6,9 @@
   <maintainer email="alex.goins@swri.org">alex</maintainer>
```

Comment:
You might change the maintainer and license while you're here.

Reply:
We can't change the license without the permission of the person who put it there (Alex).